From abe64c23743f1f6cbc1c38f27f1333ce5814c68c Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Thu, 03 Feb 2022 07:58:56 +0200
Subject: [PATCH] [SW-68803] arcboot : move bringup macro to common

The bringup defines are moved to a common header file such that it
is not restricted to usage in app/ directory. This is in preparation
to use these defines in gaudi3 specific implementations.

Change-Id: I56dde2ebab7a3b9aa7e829490887d956bed633cf
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index d96122c..838068d 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -38,9 +38,6 @@
 	},
 };
 
-#define ARCBOOT_BRINGUP_ERROR		-1
-#define ARCBOOT_BRINGUP_OK		1
-
 #define ARCBOOT_STATUS_REG_SZ_BITS		32
 __weak void arcboot_rom_status_update(enum arcboot_rom_progress progress)
 {
diff --git a/include/arcboot.h b/include/arcboot.h
index 2b7ac6f..789de6a 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -55,6 +55,8 @@
 #define COLD_RST_FLOPS_VALIDATION_PASS_MASK	0x2
 #define COLD_RST_FLOPS_EFUSE_READ_SHIFT		0x2
 #define COLD_RST_FLOPS_EFUSE_READ_MASK		0x4
+#define ARCBOOT_BRINGUP_ERROR		-1
+#define ARCBOOT_BRINGUP_OK		1
 
 #define HL_INIT_SP_SRAM_RETRIES			100
 #define HL_INIT_SP_SRAM_UDELAY			10
